### ISSUE TITLE
Latest releases not available through npm

### ISSUE THREAD

**Issue by ArturoMendes** [yamankatby/react-native-material#42]
NPM seems to have lost track of this repository (it appears to be pointing to an older one) and so new releases are not available through NPM. Thanks in advance!

**Comment by insidewill** [comment]
I also noticed this, it's almost a year old and it's not on npm

**Comment by yamankatby** [comment]
That's right, I think we have to test the changes and publish a new version. Can anyone give the changes since the latest publish a review and confirm it's alright to publish? 🫶
